.github/workflows/pull-request-checks.yaml

```diff
@@ -4,6 +4,40 @@ on:
     branches: [ develop ]
 
 jobs:
+  check-ubuntu-20_04-make-gcc:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Fetch dependencies
+        env:
+          # This is needed in addition to -yq to prevent apt-get from asking for
+          # user input
+          DEBIAN_FRONTEND: noninteractive
+        run: |
+          sudo apt-get install -yq gcc gdb g++ maven jq flex bison
+          make -C src minisat2-download
+      - name: Build with make
+        run: |
+          make -C src CXX='/usr/bin/g++' -j2
+          make -C unit CXX='/usr/bin/g++' -j2
+          make -C jbmc/src  CXX='/usr/bin/g++' -j2
+          make -C jbmc/unit CXX='/usr/bin/g++' -j2
+      - name: Run unit tests
+        run: |
+          make -C unit test
```

Comment:
I believe the unit tests, at least the jbmc ones, actually need to be run from the `unit` directory

Reply:
The `-C unit` argument to `make` changes into the `unit` directory and changes back before exiting `make`. This is the same way as the `make` build is implemented in Travis right now.